recipes/py2exe/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "py2exe" %}
+{% set version = "0.11.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/py2exe/py2exe/archive/v{{ version }}.tar.gz
+  sha256: 6bb7ff4834ef2096823710c67edf49c971422775fa41c9c7718c19f0437a2d3d
+
+build:
+  number: 0
+  entry_points:
+    - build_exe = py2exe.build_exe:main
+  skip: True  # [ not win ]
+  skip: True  # [ py < 38 ]
```

Comment:
setup.py says `>=3.6, <3.11`.

Reply:
Yup, but I noticed this:

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=447416&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=06d62fd2-ab49-5add-9ee5-83b6882d7d2b&l=1750

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=447416&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=06d62fd2-ab49-5add-9ee5-83b6882d7d2b&l=3248